generate_weekly_newsletter_json.py

```diff
@@ -19,10 +19,8 @@
     xml_gen = GenerateXML()
     elastic_search = ElasticSearchClient()
     dev_urls = [
-        ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/",
+        ["https://gnusha.org/pi/bitcoindev/",
          "https://mailing-list.bitcoindevs.xyz/bitcoindev/"],
```

Comment:
here we have an array inside an array?

Reply:
Yes, an array inside an array handle the unique mailing lists, e.g., bitcoin-dev have multiple domains but we want to show only one post related to in "Today in History", it doesn't affect the other logics such as Active Posts and Recent Posts.